.github/workflows/docker_publish.yml

```diff
@@ -40,3 +40,35 @@ jobs:
           repository: ubercadence/server
           build_args: TARGET=auto-setup
           tags: ${{ github.event.release.tag_name }}-auto-setup, latestRelease-auto-setup
+  push_server_to_ghcr:
+    name: Push Docker server image to GitHub Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Container Registry (server build)
+        if: "!github.event.release.prerelease"
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.CADENCE_SERVER_GHCR_PAT }}
+          registry: ghcr.io
+          repository: just-at-uber/cadence/server
+          build_args: TARGET=server
+          tag_with_ref: true
+  push_server_auto_setup_to_ghcr:
+    name: Push Docker server auto-setup images to GitHub Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Container Registry (auto-setup build)
+        if: "!github.event.release.prerelease"
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.CADENCE_SERVER_GHCR_PAT }}
+          registry: ghcr.io
+          repository: just-at-uber/cadence/server
```

Comment:
Need to update repository to `uber/cadence/server` once organization has been initialized with ghcr (see [here](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry/enabling-github-container-registry-for-your-organization)).